pkg/vsock/vsock_linux.go

```diff
@@ -39,6 +53,8 @@ func Dial(cid, port uint32) (Conn, error) {
 			if errno, ok := err.(syscall.Errno); ok && errno == syscall.EINTR {
 				continue
 			}
+			// Trying not to leak fd here
+			closeFD(fd)
```

Comment:
Hey, thanks for reporting this and providing a patch. 

I'd be happy to merge bar this minor nit. This ignores the return value, which is fine since we want the error from the `Connect()` call. However this may trip some linters and the canonical way would be to write:
```
_ := closeFD(fd)
```

Reply:
Thanks for the review! Good point.

`:=` gives me an error:
```
# github.com/linuxkit/virtsock/pkg/vsock
pkg/vsock/vsock_linux.go:57:6: no new variables on left side of :=
```

So I fixed it with just `_ = `.